### PR TITLE
COMUI 2476 checkbox and radio whitespace

### PR DESCRIPTION
**Pull request by conor-darcy**

### CODE REVIEW

File: packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-radio/example.html [hunk 1]
```diff
@@ -46,3 +46,9 @@ <h2>Example</h2>
     </gux-form-field-radio>
   </fieldset>
 </form>
+
+<style>
+  .example-fieldset gux-form-field-radio {
+    margin-bottom: var(--gse-core-spacing-4xs);
```
Comment (jordanstith15) [inline packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-radio/example.html:52]: so are consumers supposed to use the token themselves like this then whenever they use the component?

File: packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-checkbox/example.html [hunk 2]
```diff
@@ -77,3 +77,9 @@ <h2>Example</h2>
     Toggle Pizza field
   </button>
 </gux-button-slot>
+
+<style>
+  .example-fieldset gux-form-field-checkbox {
```
[the remark this answers is not in the document]
Comment (conor-darcy) [reply packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-checkbox/example.html:82]: I don't really like applying global CSS rules like that but I can remove it.

File: packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-radio/example.html (hunk 1 above)
Comment (conor-darcy) [reply packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-radio/example.html:52]: @jordanstith15 No, there is a radio-group component in the works which will handle the spacing. This is just there for the example.

**Comment by conor-darcy** [comment]
Closing this and replacing with https://github.com/MyPureCloud/genesys-spark/pull/415
To generate the demo url.

**Pull request closed without merging**